packages/backend/src/services/SemanticLayerService/Pivoting.ts

```diff
@@ -1,9 +1,14 @@
-import { SemanticLayerPivot, SemanticLayerResultRow } from '@lightdash/common';
+import {
+    SemanticLayerPivot,
+    SemanticLayerResultRow,
+    SemanticLayerSortBy,
+} from '@lightdash/common';
 import pl from 'nodejs-polars';
 
 export function pivotResults(
     results: SemanticLayerResultRow[],
     { values, ...options }: SemanticLayerPivot,
+    sortBy?: SemanticLayerSortBy[],
```

Comment:
Can we remove this as it is not being used? 🙇

Reply:
Yup. Wasn't sure for a bit there if we needed to keep it. Will take it out.